content/en/docs/demo/feature-flags.md

```diff
@@ -33,3 +33,17 @@ Please see the [flagd documentation](https://flagd.dev) for more information on
 how flagd works, and the [OpenFeature](https://openfeature.dev) website for more
 information on how OpenFeature works, along with documentation for the
 OpenFeature API.
+
+## Flagd configurator
+
+Flag values can be changed through the user interface provided at <http://localhost:8080/feature> when running the demo. Changing the
+values through this user interface will be reflected in the flagd service.
```

Comment:
Can we move that up?

Ideally before the sentence:

```
These flags are managed by [`flagd`](https://flagd.dev), a simple
feature flag service that supports [OpenFeature](https://openfeature.dev). Flag
values are stored in the `src/flagd/demo.flagd.json` file. To enable a flag,
change the `defaultVariant` value in the config file for a given flag to "on".
```

Maybe even get rid of the whole setence 

```
Flag values are stored in the `src/flagd/demo.flagd.json` file.
```

Reply:
I'm open to suggestions, but I'd say that it is more important to the user, how to access and change the feature flags than what file is being changed and what technology is being used.

I'd restructure to keep all the important information at the top of the page, and talk about the file and flagd at the bottom of it. As most users just want to know how to change feature flags.

But again, I'm opened to suggestions